indico/modules/categories/forms.py

```diff
@@ -92,13 +91,11 @@ class CategoryProtectionForm(IndicoForm):
     event_creation_restricted = BooleanField(_('Restricted event creation'), widget=SwitchWidget(),
                                              description=_('Whether the event creation should be restricted '
                                                            'to a list of specific persons'))
-    event_creators = PrincipalListField(_('Event creators'), groups=True, allow_external=True,
-                                        description=_('Users allowed to create events in this category'))
 
     def __init__(self, *args, **kwargs):
-        self.protected_object = category = kwargs.pop('category')
+        self.protected_object = self.category = kwargs.pop('category')
         super(CategoryProtectionForm, self).__init__(*args, **kwargs)
-        self._init_visibility(category)
+        self._init_visibility(self.category)
```

Comment:
Why do you have to create `self.category`? Is it used anywhere?

Reply:
It's accessed by one of the fields via `self.get_form().catgory`

Reply:
Ah ok. I didn't notice that.